app/main/routes_devices.py

```diff
@@ -88,6 +90,10 @@ def handle_devices():
             if uid not in active_iSpindel_sessions:
                 active_iSpindel_sessions[uid] = iSpindelSession()
             active_iSpindel_sessions[uid].alias = alias
```

Comment:
there is a slight issue in device registration given that the device type doesn't exist. I've proposed a change to your branch with this fix @cgalpin.

https://github.com/cgalpin/picobrew_pico/pull/2

Reply:
merged, thanks!